src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_INDEX = 0.75;
+    private int capacityArray = 16;
+    private MyNode<K,V> [] arrayBaskets;
+    private int size;
+
+    public MyHashMap() {
+        arrayBaskets = new MyNode [capacityArray];
+        size = 0;
+    }
+
+    private void put(MyNode<K,V> newNode) {
+        int indexBasket = Math.abs(newNode.getHashKey() % capacityArray);
```

Comment:
move this to a separate method getIndex(K key), this codes repeats several times